Source/Core/Core/HW/SI/SI_DeviceGBAEmu.cpp

```diff
@@ -0,0 +1,167 @@
+// Copyright 2021 Dolphin Emulator Project
+// Licensed under GPLv2+
+// Refer to the license.txt file included.
+
+#include <vector>
+
+#include "Common/ChunkFile.h"
+#include "Common/CommonTypes.h"
+#include "Common/Logging/Log.h"
+#include "Common/Swap.h"
+#include "Core/Core.h"
+#include "Core/CoreTiming.h"
+#include "Core/HW/GBACore.h"
+#include "Core/HW/SI/SI.h"
+#include "Core/HW/SI/SI_DeviceGBAEmu.h"
+#include "Core/HW/SystemTimers.h"
+
+namespace SerialInterface
+{
+namespace
+{
+int s_num_connected = 0;
+}  // namespace
+
+// --- GameBoy Advance "Link Cable" ---
+
+static int GetTransferTime(u8 cmd)
```

Comment:
This function should be shared with the same function in SI_DeviceGBA.cpp, which should be exposed instead of static.